org.eclipse.wildwebdeveloper/schema/json.schema.exsd

```diff
@@ -0,0 +1,107 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.eclipse.wildwebdeveloper" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="org.eclipse.wildwebdeveloper" id="org.eclipse.wildwebdeveloper.json.schema" name="Bind JSON Schema to filename pattern"/>
+      </appinfo>
+      <documentation>
+         Register JSON schema
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
+      <complexType>
+         <sequence minOccurs="1" maxOccurs="unbounded">
+            <element ref="schema"/>
+         </sequence>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+               <appinfo>
+                  <meta.attribute translatable="true"/>
+               </appinfo>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="schema">
+      <annotation>
+         <documentation>
+            URL as value
```

Comment:
It seems to me the PDE extension editor isn't really smart enough dealing with direct text values (ie the form editor doesn't suggest entries for those). Unless I'm mistaken, it's probably worth sticking to an attribute.

Reply:
Yeah, it's also not possible to set restrictions on element value. In theory I could set pattern and force valid URL in attribute.